neurolib/optimize/evolution/evolution.py

```diff
@@ -287,7 +279,16 @@ def getModelFromTraj(self, traj):
         :rtype: `neurolib.models.model.Model`
         """
         model = self.model
-        model.params.update(self.individualToDict(self.getIndividualFromTraj(traj)))
+        # resolve star notation - MultiModel
+        individual_params = self.individualToDict(self.getIndividualFromTraj(traj))
+        if self.parameterSpace.star:
+            individual_params = {
+                key_u: v
+                for k, v in individual_params.items()
+                for key_u in list(self.model.params[k.replace("STAR", "*")].keys())
```

Comment:
unfortunately, it must be there - the difference between `star_dotdict` and simple params needs to be addressed when the model is actually updated with new parameters and that is inside Evolution or Exploration..
the problem is with `namedtuple` (hence only Evolution thanks to `deap`) because you cannot have "*" inside strings that serve as the "name" in the name tuple... so I simply replace actual "*" with the word "STAR" and then I change it back... I can move the functions for this to `collections...`